resources/integration_test/testdata/adl_db_config.json

```diff
@@ -0,0 +1,38 @@
+[
```

Comment:
I decided to make the ADL storage config DATABASES field a separate file so we can more easily manipulate it when needed.